package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please populate this with a structure similar to https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/package.json

Reply:
Opened https://github.com/hashgraph/hedera-smart-contracts/pull/61 to address this.